include/libtorrent/torrent_handle.hpp

```diff
@@ -844,6 +845,8 @@ namespace aux {
 		bool need_save_resume_data() const;
 		bool need_save_resume_data(resume_data_flags_t flags) const;
 
+		add_torrent_params get_resume_data(resume_data_flags_t flags = {}) const;
```

Comment:
@arvidn 
Does it make sense to support flags in this function? The only flag that makes sense when applied to it is `save_info_dict`. But the point is that for some time now it only assign a pointer to `add_torrent_params`. What could be the problem of doing this unconditionally? The caller could just clear it if it is not needed. (IMO, `save_info_dict` could be even deprecated at all.)